packages/@tinacms/graphql/src/database/bridge/filesystem.ts

```diff
@@ -39,16 +40,16 @@ export class FilesystemBridge implements Bridge {
       return item.replace(posixRootPath, '').replace(/^\/|\/$/g, '')
     })
   }
+  public supportsBuilding() {
```

Comment:
Differentiates the FilesystemBridge, which _can_ build schemas from the GithubBridge, which doesn't need to because those files should be checked into source control (after being built via CLI)